packages/playwright-core/src/utils/dependencies.ts

```diff
@@ -53,6 +54,10 @@ export async function installDependenciesWindows(targets: Set<DependencyGroup>,
 }
 
 export async function installDependenciesLinux(targets: Set<DependencyGroup>, dryRun: boolean) {
+  const osReleaseText = await fs.promises.readFile('/etc/os-release', 'utf8');
```

Comment:
use `getUbuntuVersion()` instead

Reply:
you mean check if its `await getUbuntuVersion === ''`?

Reply:
yep!